app/celery/scheduled_tasks.py

```diff
@@ -54,6 +60,7 @@
 @statsd(namespace="tasks")
 def run_scheduled_jobs():
     try:
+        update_in_progress_jobs.delay(queue=QueueNames.JOBS)
```

Comment:
It looks like `.delay` actually does an async call to the task: https://docs.celeryq.dev/en/stable/userguide/calling.html

I think we just want to call `update_in_progress_jobs()` directly. In fact, we could just make `update_in_progress_jobs` a regular function and not a celery task.

Reply:
Sounds good! As discussed, we can call this at the start of `check_job_status` since it runs on a schedule already and the functionality of `update_in_progress_jobs` really only needs to run when we do this check, and not periodically on its own.